test/torchtext_unittest/common/torchtext_test_case.py

```diff
@@ -5,13 +5,24 @@
 import shutil
 import subprocess
 import tempfile
+from urllib.error import HTTPError
 
 import torch  # noqa: F401
 from torch.testing._internal.common_utils import TestCase
 
 logger = logging.getLogger(__name__)
 
 
+def third_party_download(test_func):
+    def inner(*args, **kwargs):
+        try:
+            return test_func(*args, **kwargs)
+        except HTTPError as e:
+            logger.warning(e)
```

Comment:
Should we be a bit more descriptive in the error message? Maybe also print out the name of the test function that is failing.

Reply:
Added!